composer.json

```diff
@@ -11,7 +11,7 @@
     "minimum-stability": "dev",
     "require": {
         "php": ">=5.3.2",
-        "symfony/event-dispatcher": "^2.0 || ^3.0"
+        "symfony/event-dispatcher": "^2.3 || ^3.0"
```

Comment:
I did testing with the code + changes to see what versions of SF would (still) work and found 2.3 the minimal. Bumping the version even higher can be done (of course) but I considered it outside of the scope of the PR.